docs/howto/bill.md

```diff
@@ -18,9 +19,29 @@ following the steps below. Your 2i2c.org email should give you access.
 
 [spreadsheet for billing]: https://docs.google.com/spreadsheets/d/1AWVCg0D_-ATub_cVsIy5XZCwqnC6uIcgwDGYK9Q7yno/edit#gid=1349808947
 
-## AWS
+## Communities in shared cloud accounts
 
-### Get costs manually via the UI (only documented option)
+The procedure for GCP projects and AWS accounts is similar. It is practically
+demonstrated in a ~20 minute [video] working in a [spreadsheet for accounting of
+cost for communities in shared clusters].
+
+The procedure from the video is:
+
+1. Open the [spreadsheet for accounting of cost for communities in shared
+   clusters].
+2. Duplicate and clear green cells a page for a previous month.
```

Comment:
I think there is a word or something missing from the sentence - it doesn't quite read correctly.

Reply:
Thank you for reviewing @sgibson91! Addressed in [358a37b](https://github.com/2i2c-org/infrastructure/pull/4023/commits/358a37ba082d5369990b2ac46ed630ca1af3427a)